pkg/controller/backup_storage_location_controller.go

```diff
@@ -95,10 +95,10 @@ func (r *BackupStorageLocationReconciler) Reconcile(req ctrl.Request) (ctrl.Resu
 			continue
 		}
 
-		log.Debug("Verifying backup storage location")
+		log.Debug("Validating backup storage location")
```

Comment:
I would bump this up to Info since it happens less frequently than the verifying message, and can be useful to know that it's happening.